components/renku_data_services/session/api.spec.yaml

```
@@ -648,6 +648,16 @@ components:
       minLength: 1
       maxLength: 99
       example: My Renku Session :)
+    BuildPlatforms:
+      description: The target runtime platforms of a session environment.
+      type: array
+      items:
+        $ref: "#/components/schemas/BuildPlatform"
+    BuildPlatform:
+      description: A runtime platform, e.g. "linux/amd64".
+      type: string
+      minLength: 1
+      maxLength: 99
```

I think the code generation can handle this. And it makes it clearer what formats we expect and what we currently support.

```suggestion
    BuildPlatform:
      description: A runtime platform, e.g. "linux/amd64".
      type: string
      enum: ["linux/amd64", "linux/aarch64"]
```